sympy/integrals/manualintegrate.py

```diff
@@ -725,10 +725,18 @@ def trig_cotcsc_rule(integral):
                 cotcsc_csceven_condition: cotcsc_csceven
             })((a, b, m, n, integrand, symbol))
 
+def trig_sindouble_rule(integral):
+    integrand, symbol = integral
+    a = sympy.Wild('a', exclude=[1/(sympy.sin(2*symbol))])
```

Comment:
@asmeurer I want to solve the integral problem in issue #12641 and I'm not sure how to match sin(2*x)  *  cos(x)/(1+cos(x)) if symbol is excluded? Thanks.